File: Domain/Model/Notfication.cs
using System;
using System.Collections.Generic;

namespace Domain;

public partial class Notfication
{
    public int NotficationId { get; set; }

    public int UserId { get; set; }

    public string Message { get; set; }

    public DateTime SentAt { get; set; }

    public int NotficationStatusId { get; set; }

    public virtual NotficationStatus NotficationStatus { get; set; }

    public virtual User User { get; set; }
}

File: frontend/src/models/notfication.ts
import { NotificationStatus } from "./notficationStatus";
import { User } from "./user";

export type Notification = {
    notificationId: number;
    userId: number;
    message: string;
    sentAt: Date;
    notificationStatusId: number;
    notificationStatus: NotificationStatus;
    user: User;
};